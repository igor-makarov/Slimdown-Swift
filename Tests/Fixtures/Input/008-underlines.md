One __two__ three _four_ five __six__ seven _eight_.