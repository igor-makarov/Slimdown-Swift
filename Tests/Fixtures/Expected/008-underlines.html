<p>One <strong>two</strong> three <em>four</em> five <strong>six</strong> seven <em>eight</em>.</p>